sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -939,14 +939,5 @@ public async Task RotateApiKey()
             credential.Update(apiKey);
             await client.DetectLanguageAsync(document);
         }
-
-        [Test]
-        public void ThrowExceptionTest()
```

Comment:
FYI: Test deleted because other test already make sure that we are returning `RequestFailedException`
See: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs#L67